default/scripting/monster_designs/ShipDesignManifest.focs.txt

```diff
@@ -0,0 +1,36 @@
+ShipDesignManifest
```

Comment:
`ShipDesignManifest,focs,txt` stores the order of the ship designs.  

Players modify it by dragging and dropping ships.